src/components/styled/Wrappers.tsx

```diff
@@ -95,6 +95,13 @@ export const SearchListItem = styled(ListItem)<ListItemProps>(({ theme }) => ({
   background: 'white',
 }));
 
+export const TaskListItem = styled(SearchListItem)<ListItemProps>(({ theme }) => ({
```

Comment:
More specialized style for items in a list of tasks because it has a new and different style (thinner border, only colored border to the left, the left border is also thinner):

Before: 
<img width="1082" height="153" alt="image" src="https://github.com/user-attachments/assets/a745cced-07c2-4355-a4c3-744664a3ffef" />

Now:
<img width="1075" height="140" alt="image" src="https://github.com/user-attachments/assets/802b11c6-a0dc-4e1b-b1c3-22e20f71c10e" />